alfresco-transform-core-aio/alfresco-transform-core-aio-boot/src/main/resources/templates/transformForm.html

```diff
@@ -73,8 +73,10 @@ <h2>Tika Test Transformations</h2>
         <h2>LibreOffice Test Transformation</h2>
```

Comment:
Will we remove top-section that is no longer needed  (as per ATS-720) ?

Reply:
I've moved the logs to the side as it might get missed if it's at the bottom and it looks out of place at the top